app/views/posts/index.html.erb

```diff
@@ -16,7 +16,7 @@
   <%= link_to "Add a new post", new_user_post_path(user_id: @user.id), class: "new-post-btn post-btn"%>
 
   <section class='post-container'>
-    <% @posts.each do |post| %>
+    <% @posts.includes(author: :comments).each do |post| %>
```

Comment:
- [ ] The n+1 query problem should be eliminated in the controller action, as that handles the business logic of the app, including queries. The views should only be used for rendering data from the controller, it should not handle queries. See an example of how you may eliminate the problem in the controller.

  ```
  @posts = @user.posts.includes(:comments)
  ```

  ![image](https://github.com/Wineshuga/pen-it/assets/81468675/3e358e0f-3fed-4904-9339-bc5a99dbb128)